src/backend/src/controllers/description-Bullets.controllers.ts

```diff
@@ -0,0 +1,36 @@
+import { Request, Response } from 'express';
+import { validationResult } from 'express-validator';
+import prisma from '../prisma/prisma';
+
+export const checkWorkPackageDescriptionBullet = async (req: Request, res: Response) => {
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    return res.status(400).json({ errors: errors.array() });
+  }
+
+  const { body } = req;
+  const { submitterId, descriptionBulletId } = body;
+
+  const descriptionId = descriptionBulletId;
+
+  const originalDB = await prisma.description_Bullet.findUnique({ where: { descriptionId } });
+
+  if (!originalDB)
+    return res
+      .status(404)
+      .json({ message: `Description Bullet with id ${descriptionBulletId} not found` });
+
+  if (originalDB.dateDeleted) {
+    return res.status(400).json({ message: 'Cant edit a deleted Description Bullet' });
+  }
+
+  const updatedDB = await prisma.description_Bullet.update({
```

Comment:
it's slightly more complex than this. if the bullet is already checked, then you need to set the usercheckedId to null and the usercheckeddate (or whatever it's called) to null as well.

the other case is that if it isn't checked, then you would also need to set the usercheckedDate to be right now (`new Date()`)